README.md

```diff
@@ -83,19 +83,20 @@ public/private key pairs that will be used to encrypt/decrypt the message):
 const keyAgreementKey = await fetchFromSomewhere();
 
 // or derive them from an existing Ed25519 signing key
-const X25519KeyPair = require('x25519-key-pair');
-const {Ed25519KeyPair} = require('crypto-ld');
-const edKeyPair = await Ed25519KeyPair.generate();
-const keyAgreementKey = X25519KeyPair.fromEdKeyPair(edKeyPair);
-// Don't forget to set your key's id. For example, DID + fingerprint
+const {X25519KeyAgreementKey2019} = require('@digitalbazaar/x25519-key-agreement-key-2019');
```

Comment:
Don't mix ESM and CommonJS, change this to `import`.

Reply:
Whoops.